website/project/model.py

```diff
@@ -890,6 +882,8 @@ def institution_relationship_url(self):
 
     def __init__(self, *args, **kwargs):
 
+        kwargs.pop('logs', [])
```

Comment:
Is this necessary? I couldn't find anywhere that we're passing `logs` into the `Node` constructor.